addons/hr_homeworking/models/hr_employee.py

```diff
@@ -44,7 +44,7 @@ def get_views(self, views, options=None):
     def _compute_name_work_location_display(self):
         dayfield = self._get_current_day_location_field()
         for employee in self:
-            employee.name_work_location_display = employee[dayfield].name
+            employee.name_work_location_display = employee[dayfield].name if employee[dayfield] else 'Unspecified'
```

Comment:
We should translate Unspecified. 🙂 

(don't forget to add it in the `.pot` file) 🙂